plugins/lookup/env_freeipa_domain.py

```diff
@@ -54,37 +55,42 @@
 
 RETURN = '''
   _list:
-    description: List of Freeipa domains for selected Environments
+    description: List of FreeIPA domains for selected Environments
     type: list
     elements: list
```

Comment:
> elements: complex

Because it returns a list if `detailed=True` or strings otherwise.

Reply:
Changed